src/mavsdk/core/mavlink_parameter_subscription.h

```diff
@@ -0,0 +1,64 @@
+//
+// Created by consti10 on 21.06.22.
+//
+
+#ifndef MAVSDK_SUPERBUILD_SRC_MAVSDK_CORE_MAVLINK_PARAMETER_SUBSCRIPTION_H_
+#define MAVSDK_SUPERBUILD_SRC_MAVSDK_CORE_MAVLINK_PARAMETER_SUBSCRIPTION_H_
```

Comment:
Use `#pragma once` instead.

Reply:
I'd say that's in theory true but not very practical.

I'm open to changing it as soon as someone requires a compiler that does not support it:
https://en.wikipedia.org/wiki/Pragma_once#Portability

Reply:
I see